app/models/user.rb

```diff
@@ -56,7 +56,13 @@ def filter(query)
 
   scope :search, -> (term) {
     search_pattern = "%#{term}%"
-    where("firstname LIKE ? OR surname LIKE ? OR email LIKE ?", search_pattern, search_pattern, search_pattern)
+    # MySQL does not support standard string concatenation and since concat_ws or concat functions do
+    # not exist for sqlite, we have to come up with this conditional
+    if ActiveRecord::Base.connection.adapter_name == "Mysql2"
+      where("concat_ws(' ', firstname, surname) LIKE ? OR email LIKE ?", search_pattern, search_pattern)
```

Comment:
I read that before... :) the root cause is because mysql treats || as the boolean logic by default. I don’t like this check we are doing but having the firstname surname as one field on the views forces as to apply this patch